### DISCUSSION TITLE
Traffic report is double what it should be

### DISCUSSION THREAD

**Discussion by baendres** [akvorado/akvorado#892] [Q&A]
I have 2 Mikrotik's setup as exporters, flows are coming in and look correct however the reported traffic is double what it should be.  The Tik's are set as 30 sampled and 30 rejected, the reported sampling rate is 2 according to the dashboard (which is what I think it should be)  the interfaces that are selected are not children or other interfaces

Can someone help me figure out or understand why this traffic isn't showing up correctly?

Thank you!

**Comment by bile0026** [comment]
If you specify the exporter IP in the filter does it show the traffic doubled still? Another thing to check is if those flows are traversing multiple interfaces on the router being reported. i.e. flow enters ether 2 and exits ether 1 and both interfaces are being sampled can lead to double traffic. Usually you'd just want to have your external interfaces sampled.

**Reply by baendres** [reply]
![image](https://github.com/akvorado/akvorado/assets/61436009/fe935627-9289-4b77-aa16-3a782176a6e7)

Traffic is still doubled when filtering by exporter address 


![image](https://github.com/akvorado/akvorado/assets/61436009/295f958a-e450-4bb5-bd70-5ff8156b41ad)

only the transit (external) interface is selected for exporting.
